symfony/config/ProjectConfiguration.class.php

```diff
@@ -25,7 +25,7 @@ public function setup()
     if (file_exists($resourceIncFile)) {
         require_once $resourceIncFile;
     } else {
-        sfConfig::set('ohrm_resource_dir', sfConfig::get('sf_web_dir'));

```

Comment:
Who this resource dir create again? via publish-assets?

Reply:
yeah. Only this fix valid for the login page. All other internal screens are failing if publish-assets not ran.